specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/virtualMachine.json

```diff
@@ -2732,6 +2732,10 @@
           "type": "boolean",
           "description": "Specifies whether the network interface is accelerated networking-enabled."
         },
+        "disableTcpStateTracking": {
```

Comment:
I see you are defining this in this file and also defining it twice in the vmss.json
Let's define it once in ComputeRpCommon.json and refer to that definition instead

Reply:
nevermind this is a subproperty not an object so this is good